plugins/domains/scribd.com/scribd.com-error.js

```diff
@@ -1,14 +1,48 @@
+const URL = require("url");
+
 module.exports = {
 
     re: require('./scribd.com').re,
 
-    getData: function(oembedError, options, cb) {
-        if (oembedError === 401){
-            cb(null, {
-                message: "Scribd doesn't support embedding of private documents"
-            })
+    mixins: [ "*" ],
```

Comment:
Let's remove `mixins: [ "*" ],` from here and place it instead of all mixins in the main plugin.